src/daemon/netlink.c

```diff
@@ -28,6 +28,17 @@
 
 #define NETLINK_BUFFER 4096
 
+#ifndef DEFAULT_RTNL_BUFSIZE
+#define DEFAULT_RTNL_BUFSIZE	256 * 1024
+#endif
+
+#ifndef RTNL_SND_BUFSIZE
+#define RTNL_SND_BUFSIZE	DEFAULT_RTNL_BUFSIZE
+#endif
+#ifndef RTNL_RCV_BUFSIZE
+#define RTNL_RCV_BUFSIZE	DEFAULT_RTNL_BUFSIZE
+#endif
```

Comment:
I would define all that directly from `configure.ac`. I can do that for you. The default is the value of `net.core.rmem_default`? So, with the value you specified, it seems to match the current default.